src/components/ArticleDisplay.jsx

```diff
@@ -0,0 +1,14 @@
+function ArticleDisplay({article}) {
+    console.log(article)
+    return (
+      <article className='article'>
+          <img id='icon-2' src={article.article_img_url} alt="Football Image" />
```

Comment:
can you think of a way to make the alt tag dynamic as it won't always be a football image